src/mame/drivers/savquest.cpp

```diff
@@ -111,7 +144,7 @@ class savquest_state : public pcat_base_state
 
 	uint8_t m_mtxc_config_reg[256];
 	uint8_t m_piix4_config_reg[8][256];
-	uint32_t m_pci_3dfx_regs[0x40];
+	uint32_t m_pci_3dfx_regs[0xff]; // provide 256 bytes of configuration registers
```

Comment:
you provided 255

Reply:
Indeed, well spotted. The PCI specification states that a 256 bytes configuration register should be provided. But 3Dfx happened to make up their own rules as far as that specification goes. There are actually 2 large blocks reserved which we could step over to reduce the space required. 

6.  PCI Configuration Register Set
Reserved  20-59 (0x14-0x3b)  
Reserved  88-255 (0x58-0xff) 

https://web.archive.org/web/20210803204742/https://www.voodooalert.de/de/files/whitepaper/voodoo2.pdf